src/api/scan/checker.go

```diff
@@ -24,6 +25,9 @@ import (
 
 // Checker checker which can check that the artifact is scannable
 type Checker interface {
+	// HasCapability returns true when the artifact supported by the scanner
```

Comment:
I find the capability .vs. scannable a little confusing.

Reply:
Removed